src/main/java/bio/terra/service/profile/ProfileService.java

```diff
@@ -122,10 +122,13 @@ public String updateProfile(
    * </ul>
    *
    * @param id the unique id of the bill profile
+   * @param deleteCloudResources flag to also delete azure cloud resources such as storage account
+   *     and monitoring resources
    * @param user the user attempting the delete
    * @return jobId of the submitted stairway job
    */
-  public String deleteProfile(UUID id, AuthenticatedUserRequest user) {
+  public String deleteProfile(
+      UUID id, Boolean deleteCloudResources, AuthenticatedUserRequest user) {
```

Comment:
At this point, we know that `deleteCloudResources` is non-null, so it should be declared as `boolean`.
```suggestion
      UUID id, boolean deleteCloudResources, AuthenticatedUserRequest user) {
```